pyproject.toml

```diff
@@ -24,9 +24,14 @@ texttable = "^1.6.2"
 PyJWT = "^2.1.0"
 webexteamssdk = "^1.3"
 slack-sdk = "^3.4.2"
+celery = [
```

Comment:
Will this work with just running an update instead of pinning versions?

Reply:
No.  There are issues with 5.2.3 and setuptools, released on 12/29/2021.  See release notes [here](https://github.com/celery/celery/releases/tag/v5.2.3).

In it, they pin the version of setuptools, which breaks compatibility with Nautobot.  There are also Markdown related version issues, which is required to be set to !=3.3.5 to also prevent dependency issues.